### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by bajajcodes**
### Avatar Component

- avatar property border-radius will be 50% or full to give round shape.
- avatar is available in six sizes xs, sm, md, lg, xl, and 2xl.
- avatar component uses utility border for adding border to avatars, default border-color will be black and six more border colors can also be added.

127 0 0 1_5500_css_components_component html (1)

### CODE REVIEW

File: css/components/component.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="avatar.css">
+    <title>Component: Avatar</title>    
+    <style>
+        div{
```
Comment (bhanuprakashj) [inline css/components/component.html]: do not mix html and css unless it is highly necessary

Comment (bajajcodes) [reply css/components/component.html]: Hey @bhanuprakashj that file is for `preview-only` and I missed to remove before push.

**Review by ritik-patel05 — COMMENTED** [review]
for avatar-xs/sm/md/lg/xl/2xl and border sizing.
You can use variables for width and height. So that in the future, just changing those values will work.

Suggestion: Create variables for all the units for sizing, and use them in your files. It Will help in consistency and reusability.

**Comment by bajajcodes** [comment]
### Merged avatar component to dev branch

- Prefer short hand properites only.
- Implemented @ritik-patel05 suggestion to create variables for all the units for sizing.
- Using rem unit and calc function to calculate sizes of avatar components keeping ration in between sizes to 1.5 and minimum size will be 3 x1rem and maximum will be 10.5 x 1rem.
- Mantain consistency and reusability.

**_`Design wins over Development, prioritize design over development`_**

**Pull request merged**